src/main/scala/com/codacy/api/client/CodacyClient.scala

```diff
@@ -96,9 +96,8 @@ class CodacyClient(
       parseJsonAs[T](body)
     } catch {
       case NonFatal(ex) =>
-        val zero: Option[Int] = Some(0)
-        (noRetries, zero) match {
-          case (Some(noRetries), Some(zero)) if noRetries.>(zero) =>
+        (noRetries, Some(0)) match {
+          case (Some(noRetries), Some(0)) if noRetries.>(0) =>
             post(request, value, timeoutOpt, sleepTime, Option(noRetries.-(1)))
```

Comment:
In Scala you can call methods in two ways.
With dot notation:
```scala
a.+(b)
```

With space notation:
```scala
a + b
```

And this applies for every type of method with a single parameter:

```scala
@ "codacy.com".split('.')
res0: Array[String] = Array("codacy", "com")

@ "codacy.com" split '.'
res1: Array[String] = Array("codacy", "com")
```

In case of symbolic methods like `>`, `-` etc. the best practice is to use the spaced notation.
Also when you're wrapping a non-nullable thing (like a number).
```suggestion
          case (Some(noRetries), Some(0)) if noRetries > 0 =>
            post(request, value, timeoutOpt, sleepTime, Some(noRetries - 1))
```